lib/utils.js

```diff
@@ -219,9 +224,12 @@ const utils = {
    */
   getScopeForPr (pr) {
     const matches = pr.description.match(/#[A-Za-z]+#/g)
+    const prLink = `[PR #${pr.number}](${pr.url})`
 
     if (!matches) {
-      throw new Error(`No version-bump scope found for PR #${pr.number} (${pr.url})`)
+      const example = 'Please include a scope (i.e. `#major`, `#minor#`, `#patch#`) in your PR description.'
+      const exampleLink = 'See https://github.com/ciena-blueplanet/pr-bumper#pull-requests for more details.'
+      throw new Error(`No version-bump scope found for ${prLink}\n${example}\n${exampleLink}`)
```

Comment:
@gknoy How's this for a new and improved error message when missing the PR scope?